sonar-xml-plugin/src/test/java/org/sonar/plugins/xml/checks/AbstractCheckTester.java

```diff
@@ -50,6 +50,9 @@
 
   static final String INCORRECT_NUMBER_OF_VIOLATIONS = "Incorrect number of violations";
 
+  /**
+   * Intended for testing.
```

Comment:
@pynicolas You're right. I thought it was a main class, but it is a test class indeed.